src/test/java/by/onliner/tests/LogInTests.java

```diff
@@ -0,0 +1,89 @@
+package by.onliner.tests;
+
+import by.onliner.AbstractTest;
+import by.onliner.core.anotation.TestType;
+import io.qameta.allure.Feature;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import static by.onliner.constants.OnlinerConstants.*;
+import static by.onliner.constants.TestType.SMOKE;
+
+@Feature("Log-In")
+public class LogInTests extends AbstractTest {
+
+    @TestType(value = SMOKE)
+    @Test(description = "[Test-Case ID:ONL_006] Test for log-In when fields are empty")
+    public void login_whenFieldsAreEmptyTest() {
+        mainPage.openPage();
+        mainPage.getMenu().openLogInPage();
+
+        logInPage.clickLogInFormButton();
+
+        Assert.assertTrue(logInPage.isLogInPageDisplayedAfterTest(), "Log-In page is not displayed after test");
+        String actualLogInWarningText = logInPage.getInputFieldWarningText();
+        Assert.assertEquals(actualLogInWarningText, INCORRECT_LOG_IN_WARNING_TEXT, String.format("The actual log-in " +
+                "warning text [%s] don't much expected warning text [%s]", actualLogInWarningText, INCORRECT_LOG_IN_WARNING_TEXT));
+    }
+
+    @TestType(value = SMOKE)
+    @Test(description = "[Test-Case ID:ONL_007] Test for log-In when password field is empty")
+    public void login_whenPasswordFieldIsEmpty() {
+        mainPage.openPage();
+        mainPage.getMenu().openLogInPage();
+
+        logInPage.inputLogIn(LOGIN);
+        logInPage.clickLogInFormButton();
+
+        Assert.assertTrue(logInPage.isLogInPageDisplayedAfterTest(), "Log-In page is not displayed after test");
+        String actualPasswordWarningText = logInPage.getInputFieldWarningText();
+        Assert.assertEquals(actualPasswordWarningText, INCORRECT_PASSWORD_WARNING_TEXT, String.format("Actual password " +
+                "warning text [%s] don't much expected warning text [%s]", actualPasswordWarningText, INCORRECT_PASSWORD_WARNING_TEXT));
+    }
+
+    @TestType(value = SMOKE)
+    @Test()
+    public void login_whenAfterWrightLoginPutFewGaps() {
+        mainPage.openPage();
+        mainPage.getMenu().openLogInPage();
+
+        logInPage.inputLogIn(LOGIN_WITH_FEW_GAPS);
```

Comment:
Store test data which is related to current tests and not to other tests (i don't think you are going to reuse invalid credentials for other tests) in the class with the tests

Reply:
Fixed